src/pages/CommandView/form-data/get-model/get-model.ts

```diff
@@ -0,0 +1,126 @@
+import { Parameter } from 'types/backend-types'
+
+type ParameterModel = {
+  [key: string]: string | number | boolean | object | null
+}
+
+const mergeModels = (a: object, b: object) => {
```

Comment:
It might "work", in the JavaScript sense, but I think it wouldn't type-check. I end avoiding a lot of the old JavaScript patterns just for that reason.